try() function is used outside function
